CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ Notable changes to this project are documented in this file. The format is based
 
 ## [Unreleased]
 
+Breaking changes:
+
+New features:
+
+Bugfixes:
+
+Other improvements:
+
+## [v9.0.0](https://github.com/purescript-contrib/purescript-parsing/releases/tag/v9.0.0) - 2022-04-27
```

Comment:
We may need to remember to update this date if this release goes out later than today.

Reply:
Ah, good point.